monkey/monkey_island/cc/models/errors.py

```diff
@@ -0,0 +1,2 @@
+class MonkeyNotFoundError(Exception):
```

Comment:
Why do you need this class here? Isn't this dir for defining documents?

Reply:
Upon second reading, having this error be defined in a separate file then the method that's throwing it is confusing at best and annoying at worst. I've moved it to the `monkey.py` model file. Thanks!